Chapter_2_Getting_Started.md

```diff
@@ -0,0 +1,1206 @@
+> LO-2.1.2.4 Understand the purpose and limitations of the `*** Keywords ***` section. (K2)
+
+This section allows you to define **locally scoped user keywords** that can only be used within this suite where they are defined,
+while keywords defined in resource files can be used in any suite that imports these resource files.
+Keywords defined in a suite are therefore not reusable outside the suite,
+but they are often used to organize and structure tests|tasks for improved readability and maintainability.
+This section is particularly useful for defining suite-specific actions,
+such as **Suite Setup** keywords or similar kinds,
+which are relevant only to the suite they belong to.
+
+While these keywords are not globally accessible,
+they serve a crucial role in making the suite more modular
+and understandable by breaking down complex sequences into smaller, manageable parts.
+Defining keywords locally in this section enhances the maintainability of the tests|tasks within the suite,
+ensuring that even large and intricate suites remain well-structured and easy to manage.
+
+See [3.3.1 `*** Keywords ***` Section](Chapter_3_Keyword_Design_Variables_Resources.md#331--keywords--section) for more information about the `*** Keywords ***` section.
+
+
+#### 2.1.2.5 `*** Comments ***` Section
+
+This section is used to add comments to the suite file or resource file.
+All content in this section is ignored by Robot Framework and is not executed or parsed.
+
+
+
+
+## 2.2 Basic Suite File Syntax
+
+<!--
+TODO:
+I think this section needs a bit more structure and we should introduce the concept of "settings" like [Documentation] already here and reference to Chapter 4.
+
+ -->
+
+> [!IMPORTANT]
+> LO-2.2 Understand the basic syntax of test cases and tasks. (K2)
+
+
+
+### 2.2.1 Separation and Indentation
+
+> [!IMPORTANT]
+> LO-2.2.1 Understand and apply the mechanics of indentation and separation in Robot Framework. (K3)
+
+As mentioned before, Robot Framework uses an indentation-based and space-separated syntax to structure keywords, test cases, and tasks.
+
+**Two or more spaces** are used to separate or indent statements in Robot Framework files, while a single space is a valid character in tokens (i.e. keyword names, argument values, variables, etc.).
+The clear recommendation for separators is to use **four spaces** or more to unambiguously make it visible
+to a potential reader where elements are separated or indented.
+
+A statement in Robot Framework is a logical line that contains specific data tokens which are separated by multiple spaces (separator token) from each other.
+
+**Example 1**: A keyword call is a statement that consists of a keyword name and its arguments, which are separated by two or more spaces from the keyword name and from each other.
+An optional assignment of the return value can be possible as well.
+The line comments starting with a hash `#` show the tokens in the statement.
+
+Example with tokens in comments:
```

Comment:
The example introduces continuation `...`, but it is not explained beforehand nor immediately afterwards.

Reply:
fixed